qa/rpc-tests/wallet_shieldingcoinbase.py

```diff
@@ -264,7 +265,7 @@ def run_test (self):
         # given the tx size, resulting in mempool rejection.
         errorString = ''
         recipients = []
-        num_t_recipients = 2500
+        num_t_recipients = 1998 # stay just under the absurdly-high-fee error
```

Comment:
Is this comment still accurate?

Reply:
Yes – going above this hits the absurd check on the mining side. The test later in this file that does `WEIGHT_RATIO_CAP * conventional_fee(num_t_recipients) + 0.0000001` exceeds that, but hits the tx-creation-side absurd check before it gets to the mempool.